docs/.gitignore

```diff
@@ -39,4 +39,7 @@ jspm_packages
 
 install-state.gz
 out
-_pagefind
\ No newline at end of file
+_pagefind
+
+# Ignore yarn cache artifacts
+.yarn/cache
```

Comment:
According to [the Slack discussion](https://input-output-rnd.slack.com/archives/C07FD0DNEMQ/p1744373970386369?thread_ts=1744228336.979659&cid=C07FD0DNEMQ), we should not ignore `.yarn/cache`, see https://yarnpkg.com/features/caching#zero-installs

(I'm not saying that I agree with this approach, but it’s already there.)